src/e2e-app/main-aot.ts

```diff
@@ -0,0 +1,4 @@
+import {platformBrowserDynamic} from '@angular/platform-browser-dynamic';
```

Comment:
Instead of having a separate `main-aot.ts`, we should just make this the main `main.ts`; nobody should need to ever run this in JIT mode